### DISCUSSION TITLE
No UI feedback on IOS when app is opened and “explore” page is loaded

### DISCUSSION THREAD

**Discussion by jusso-dev** [community/community#64398] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</
When the iOS app is opened, there is no UI feedback that the app is loading, a simple ghost list view or spinner would suffice.
summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone15,3 (iOS 17.0)
</details>

**Comment by Iron-Ham** [comment]
Hi @jusso-dev 

Thank you for your feedback. 
Can you provide us with additional information:
- What behavior are you expecting to occur?
- What behavior is occurring? 

If you could provide a screenshot or video, that would be most helpful! 


Thank you,
Hesham

**Reply by jusso-dev** [reply]
Thanks @Iron-Ham 

I've attached some screenshots upon opening the app.

I'm expecting as an end user to be given some feedback that the app is in fact fetching data from a server, as a dev I know this is happening, but to those less technical it simply appears as if the app is not working. 

When opening app, blank list is present 
![image](https://github.com/community/community/assets/16183422/3552b8ec-4ba1-49cf-a0df-dca6cba39c20)

After 10 seconds or so, list is rendered:
![image](https://github.com/community/community/assets/16183422/cdcf9540-0840-4d2c-9a7d-376276f57f66)

**Reply by Iron-Ham** [reply]
Thanks for clarifying. It seems that our empty state is appearing when it shouldn't, and as a result is preventing the loading spinner from displaying. 

I'll file a bug, and make sure that I (or someone else on the team) takes a look at it.

Thanks @jusso-dev!
